### PR TITLE
sql: Add `Delete` and `Update` in DML doc

### PR DESCRIPTION
**Pull request by shenli**
Document for `Delete` and `Update` statements.
`Replace` statement will be in the next PR.

### CODE REVIEW

File: sql/dml.md [hunk 1]
```diff
@@ -3,12 +3,20 @@ title: 数据操作语言
 category: user guide
 ---
 
+
```
Comment (queenypingcap) [inline sql/dml.md]: Please remove the redundant line.

**Comment by shenli** [comment]
@queenypingcap PTAL

**Comment by shenli** [comment]
@sunhao2017

File: sql/dml.md [hunk 2]
```diff
@@ -139,4 +146,98 @@ CREATE TABLE tbl_name1 (
...
+    FROM table_references
+    [WHERE where_condition]
+
+DELETE [LOW_PRIORITY] [QUICK] [IGNORE]
+    FROM tbl_name[.*] [, tbl_name[.*]] ...
+    USING table_references
+    [WHERE where_condition]
+```
+
+删除多个表的数据的时候，可以用这两种语法。这两种写法都可以指定从多个表查询数据，但只删除其中一些表的数据。在第一种语法中，只会删除 `FROM` 关键字之前的 Table 列表中所列 Table 的表中的数据。对于第二种写法，只会删除 `FROM` 之后 `USING` 之前地 Table 列表中的所列 Table 中的数据。
```
Comment (andelf) [inline sql/dml.md]: 地-> 的

File: sql/dml.md [hunk 3]
```diff
@@ -139,4 +146,98 @@ CREATE TABLE tbl_name1 (
...
+
+# Update 语句
+
+Update 语句用于更新表中的数据。
+
+## 语法定义
+
+Update 语句一共有两种语法，分别用于更新单表数据和多表数据。
+
+###  单表 Update
```
Comment (andelf) [inline sql/dml.md]: 2 spaces

**Comment by shenli** [comment]
@andelf PTAL

**Pull request merged**